### PR TITLE
Correctly compute project SHA when Manifest contains paths starting with `../`

### PR DESCRIPTION
**Pull request by tpgillam**
Fixes https://github.com/julia-vscode/SymbolServer.jl/issues/285

This PR ensures that the project SHA is correct computed when an environment's Manifest contains relative paths starting with `../`.

It also adds a test that would have caught the issue above.

To the reviewer: I bumped the patch version, since I'm used to releasing on every PR, but do advise what would be standard. (Naturally we're keen for the issue to be resolved in LanguageServer soon!)

---

For every PR, please check the following:
- [x] End-user documentation check. If this PR requires end-user documentation in the Julia VS Code extension docs, please add that at https://github.com/julia-vscode/docs.
- [ ] Changelog mention. If this PR should be mentioned in the CHANGELOG for the Julia VS Code extension, please open a PR against https://github.com/julia-vscode/julia-vscode/blob/master/CHANGELOG.md with those changes.

### CODE REVIEW

**Review by pfitzseb — DISMISSED** [review]
LGTM

**Comment by pfitzseb** [comment]
Let's disable nightly CI temporarily so we can get this merged (unless you wanna debug and fix that failure too).

**Comment by tpgillam** [comment]
> Let's disable nightly CI temporarily so we can get this merged (unless you wanna debug and fix that failure too).

Ha, I think I'll pass on looking at Julia nightly builds -- but why don't I add a commit just to temporarily disable that one? Then we can convince ourselves that nothing is broken at least.

**Review by pfitzseb — APPROVED** [review]

File: .github/workflows/jlpkgbutler-ci-pr-workflow.yml [hunk 1]
```diff
@@ -9,7 +9,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.0', '1.6', '1.9', 'nightly']
+        # FIXME: temporarily removing nightly due to failing build.
```
Comment (tpgillam) [inline .github/workflows/jlpkgbutler-ci-pr-workflow.yml:12]: Should we revert this before merging, now that the tests have passed?

Comment (pfitzseb) [reply .github/workflows/jlpkgbutler-ci-pr-workflow.yml:12]: Yeah, let's.

Comment (tpgillam) [reply .github/workflows/jlpkgbutler-ci-pr-workflow.yml]: Done.

**Pull request merged**